src/ripple/app/main/Main.cpp

```diff
@@ -150,7 +150,7 @@ void printHelp (const po::options_description& desc)
            "     version\n"
            "     server_info\n"
            "     sign <private_key> <json> [offline]\n"
-           "     sign_for\n"
+           "     sign_for <signer_address> <signer_private_key> <json> [offline]\n"
            "     stop\n"
            "     submit <tx_blob>|[<private_key> <json>]\n"
            "     submit_multisigned\n"
```

Comment:
I recommend
`submit_multisigned <tx_json>`

While you're at it, you could also modify the sign and submit usage messages to use <tx_json> instead of <json>